src/lein_monolith/task/fingerprint.clj

```diff
@@ -325,14 +337,16 @@
    ::sources
    ::deps
    ::java-version
-   ::upstream])
+   ::upstream
+   ::jar-exclusions])
 
 
 (def ^:private reason-details
   {::up-to-date ["is up-to-date" "are up-to-date" :green]
    ::new-project ["is a new project" "are new projects" :red]
    ::version ["has a different version" "have different versions" :red]
    ::java-version ["has a different java version" "have different java versions" :red]
+   ::jar-exclusions ["has different JAR exclusions" "have different JAR exclusions" :red]
```

Comment:
This could probably be yellow instead of red. No hard and fast rules here, but jar exclusions aren't _guaranteed_ to change the behavior of a project, which makes them similar to the seed, dependencies, upstream fingerprints, etc.
```suggestion
   ::jar-exclusions ["has different JAR exclusions" "have different JAR exclusions" :yellow]
```